util/tooltool/tooltool.py

```diff
@@ -361,7 +358,7 @@ def list_manifest(manifest_file):
     try:
         manifest = open_manifest(manifest_file)
     except InvalidManifest as e:
-        log.error("failed to load manifest file at '%s': %s" % (
+        log.error("failed to load manifest file at '%s': %s", (
```

Comment:
The brackets should not be around the args following comma (`"" % (a1, a2)` vs. `log.error("", a1, a2)`)